notebooks/notebooks/iclr2022/winrate-predictions.ipynb

```diff
@@ -0,0 +1,338 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #26.    fig = plt.figure(figsize=(2.6448 + 0.15, 2.6448), dpi=150)
</pre><p>I think with the <code>plt.style.use()</code> in the first cell that you shouldn't need to specify the figsize &amp; DPI anymore, unless you don't like the figure height given by the style</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/HumanCompatibleAI/KataGoVisualizer/pull/20/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
I want the manual figsize but yes I think dpi is superfluous

Reply:
<p>Any reason you're not just making your own style sheet? Having figure size hard-coded has a number of disadvantages (duplicated between two cells; will require code change if we want figure in different style for different purpose like a presentation).</p><p> </p><p>Separately I tend to think the figures look a bit too tall right now, although the default height (2 inches) might well be too small for them.</p>